org.audacious_media_player.Audacious.yaml

```diff
@@ -0,0 +1,165 @@
+# WIP
+id: org.audacious_media_player.Audacious
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: audacious
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-music
+  - --own-name=org.atheme.audacious
```

Comment:
Flatpak automatically grants DBus permissions based on the app id, so I'd say that's how the app should be named.

Reply:
Yep. But until upstream will rename it we need this or app will not even launch.

Reply:
@tim77 Can you explain this more? What exactly do you want renamed, upstream?

Reply:
@jlindgren90 if https://atheme.org/ is still actual then nothing for sure. It's just looking non-intuitive and thought maybe https://audacious-media-player.org/ is actual one now domain and name. So lets stick with `org.atheme.audacious` right? Or maybe at least with uppercase for app name as now recommended? `org.atheme.Audacious`

Reply:
> @tim77 Can you explain this more? What exactly do you want renamed, upstream?

To make things clearer: the current application uses `org.atheme.audacious`. Flathub requires having the ownership of the reverse domain name used as an application ID. What was suggested is using `org.audacious_media_player.Audacious` as an application ID

Reply:
Audacious used to be part of https://atheme.org but there's no longer any official relationship. https://audacious-media-player.org is the current and official website. The D-Bus API still contains "org.atheme" because it's established API. I don't know if it's more important for the flatpak name to match the website or the D-Bus ID, since they are (and will remain) different.

Reply:
Now all plugins built. So as for AppID, we shall refer to:

1. `org.atheme.audacious`
2. `org.atheme.Audacious`
3. `org.audacious_media_player.Audacious`

?

Reply:
it's more important to match the existing dbus api - it's causes you real pain if it doesn't.

Reply:
Switched to `org.atheme.audacious`.